sources/packages/api/src/database/entities/offering-status.type.ts

```diff
@@ -1,16 +1,22 @@
 /**
  * Represents the status of an offering.
+ ** Offering statuses are grouped into the ones which are exclusively for offering creation
```

Comment:
Thanks for adding the comments. I would not abuse the double starts comments `**`.
I still believe that the individual enum items can be commented on. Not a blocker.